vortex-array/src/lib.rs

```diff
@@ -4,9 +4,10 @@
 #![feature(trusted_len)]
 #![feature(array_chunks)]
 #![feature(iterator_try_collect)]
+#![feature(associated_type_defaults)]
```

Comment:
is this still needed?

Reply:
Nope. I found it was too easy to forget to override.